purchase_request_department/models/purchase_request.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2017-2020 Forgeflow S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+from odoo import api, fields, models
+
+
+class PurchaseRequest(models.Model):
+    _inherit = "purchase.request"
+
+    def _get_my_department(self):
+        employees = self.env.user.employee_ids
+        return (
+            employees[0].department_id
+            if employees
+            else self.env["hr.department"] or False
+        )
+
+    department_id = fields.Many2one(
+        "hr.department", "Department", default=_get_my_department
```

Comment:
Non-blocking. Just a bit more declarative for the future :), 
```suggestion
        comodel_name="hr.department", string="Department", default=_get_my_department
```

Reply:
I think I read somewhere that using lambda is more preferable, so your function can be located under the fields too.
```suggestion
        "hr.department", "Department", default=lambda self: self._get_my_department(),
```